pkg/cmd/kubernetes_exec.go

```diff
@@ -227,7 +228,8 @@ func helmInit() error {
 	return nil
 }
 
-func kubectl(kubeConfigPath string, kubeContext string, parts ...string) execute.ExecTask {
+func kubectl(command *cobra.Command, parts ...string) execute.ExecTask {
```

Comment:
@alexellis the important change is here can we agree on this? Then I will update the fork with the changes since it seems that I most likely will have do the cleanup again.